lib/main_common.pm

```diff
@@ -2058,6 +2059,30 @@ sub load_virtualization_tests {
     return 1;
 }
 
+sub load_xen_tests {
+    return unless check_var('HOST_HYPERVISOR', 'xen');
```

Comment:
What's about reusing `load_autoyast_tests` and adjusting it for xen?

Reply:
My current tests are not about autoyast, installation via autoyast is only an option (and currently I even do not have the correct profile). Adjusting ``load_autoyast_tests`` would mean to add options to that function to skip autoyast and install xen manually. I do not think we want this.

Reply:
Sorry, I confused the names, I meant `load_default_autoyast_tests` which is in `products/sle/main.pm`. It cannot be simply reused, but will decrease code duplication, including integrating `virt_autotest/reboot_and_wait_up_normal` intoload_boot_tests load_reboot_tests, so we can abstract from backend on a level what we expect from the system.

Reply:
Hm, yes, that might be a good improvement. However it will need some improvements to adapt the backend dependent behavior from the (installation) tests.